### DISCUSSION TITLE
filament runout sensor installed but nozzle keeps parking wen printing

### DISCUSSION THREAD

**Discussion by wazana1979** [Jyers/Marlin#470] [Q&A] [answered]
hello i have tryed the Smith3D_E3V2-2.0.x.17-SLOW 5X5  as well as the new V1.3.0 - Jyer build (E3V2-BLTouch-3x3-v4.2.2 firmware. every time i enable the filament runout sensor in the menu the nozzle goes to park afterr 2 seconds of printing. the filement runout sensor is a swich sensor and my runout distance is set to 25.0
any assistance would be wonderfull

**Comment by Jyers** [comment, marked as answer]
Most likely your runout sensor is opposite to that which is set by default in the firmware. You would need to compile your own version with FIL_RUNOUT_STATE set to HIGH.

**Reply by wazana1979** [reply]
Is there a precompiled version ? Idealy with 5x5 slow. I am new to this and have no coding experiance

**Reply by Jyers** [reply]
Unfortunately I can't make precompiled files for both versions as it would overload my releases. I can recommend looking into compiling your own as there are a lot of resources online and its not as hard as it seems. However, I think I'll open up a discussion thread to ask peoples runout sensor version so I can make the default the most popular one.

**Reply by wazana1979** [reply]
jyers,
i have managed to compile the firmware thanks for the assistance i really like this new firmware :)

**Reply by natrixmusic** [reply]
I have the same problem as you but when i compile the firmware i don't have all options onder the advance menu

**Reply by Jyers** [reply]
What options are missing?
